entity-services-examples/src/main/java/com/marklogic/entityservices/examples/person/PersonExplorer.java

```diff
@@ -187,7 +193,7 @@ public void runOpticQuery() {
 //                .select(pb.col("firstName"));
 
         Plan p = pb.fromTriples(pb.pattern(pb.subjects(pb.col("s")),
-            pb.predicates(pb.sem.iri("http://www.w3.org/1999/02/22-rdf-syntax-ns#")),
+            pb.predicates(pb.sem.iri("http://www.w3.org/1999/02/22-rdf-syntax-ns#type")),
```

Comment:
Good question.  without the 'type' here it's a bug.  Adding 'type' turns this IRI from garbage into rdf:type, which is what we wanted to query for.